lib/apiservers/engine/backends/container.go

```diff
@@ -1315,6 +1316,8 @@ func (c *Container) ContainerChanges(name string) ([]docker.Change, error) {
 			change.Kind = docker.ChangeAdd
 		case "D":
 			change.Kind = docker.ChangeDelete
+			p := strings.TrimPrefix(filepath.Base(change.Path), docker.WhiteoutPrefix)
+			change.Path = filepath.Join(filepath.Dir(change.Path), p)
```

Comment:
cool, we could always attempt the strip off of "/". I think it should be fine though. I have seen this show up in weird ways for other parts of VIC. So I just wanted to point this out.